src/engine.single.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from "chai";
+import Engine from "..";
+import { AssertionError } from "assert";
+import { Player } from "./enums";
+
+describe("Engine", () => {
```

Comment:
If you want to run only a single test, there's no need to make a separate file. You can just do `describe.only` or `it.only`. Then `npm test` will only run that single test.

Reply:
ok, thank you